tests/components/mqtt/test_lock.py

```diff
@@ -34,6 +34,8 @@
                 "command_topic": "command-topic",
                 "payload_lock": "LOCK",
                 "payload_unlock": "UNLOCK",
+                "state_locked": "LOCKED",
```

Comment:
We should add a test where we change the state value to something else than the default config.

Reply:
Just added two test, they should complete with the next pipeline